react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/FontAssetManager.kt

```diff
@@ -0,0 +1,53 @@
+package com.revenuecat.purchases.react.ui
+
+import android.content.res.AssetManager
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import com.facebook.infer.annotation.Nullsafe
+
+@Nullsafe(Nullsafe.Mode.LOCAL)
+object FontAssetManager {
+    private val mFontCache: MutableMap<String?, List<Font>?> = hashMapOf()
```

Comment:
Should the key and value be nullable? I believe we could make them not nullable and only have a key when there is a value.